lib/superstring/index.js

```diff
@@ -1,6 +1,9 @@
 var _ = require('../util').lodash,
     uuid = require('uuid'),
+    faker = require('faker'),
```

Comment:
This will include all locale data, to include specific set of locales:

```suggestion
    faker = require('faker/locale/en'),
```